_posts/research/2020-04-22-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,95 @@
+description: Multi-Agent Reinforcement Learning for Robot Control
+author: Joewie J. Koh and Guohui Ding
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Heterogeneous teams of cooperative multi-robot systems have the potential to propel robotics at large toward a safer, more accessible, and lower cost future.
+Through distribution of roles among the various agents, a multi-robot system can be imbued with capabilities that are more than the sum of its parts by coordinating multiple agents to cooperatively accomplish common goals; using search and rescue as an example, cooperation between unmanned aerial and ground vehicles could increase the odds of success with larger areas covered, better mobility.
+However, autonomous coordination of robots cannot rely on predefined rules or simple control schemes.
+Robots are both generators and consumers of data, and appropriate decision-making models that map observed data to actions can endow a multi-robot system with intelligent behaviors that improve cooperation.
+Our work proposes several approaches to fully leverage multi-agent RL for autonomous robot coordination.
+
+<!-- More -->
+
+{% include image.html url="research/multirobot/cooperative_manipulation.png" max-width="75%" description="An object too heavy or unwieldy for a single robotic arm could be manageable for two cooperating robots. Robots cooperating as needed could increase the efficiency of robotic systems, as the robots can work on independent tasks at other times." %}
+
+# Challenges
+
+One of the main challenges in intelligent robotics is interaction with the environment to achieve a goal, which motivates the question of "how a robot should learn to manipulate the world around it" [1].
+As a decision-making model, reinforcement learning (RL) has been widely studied in the multi-agent context because of its performance in a broad range of applications.
+Compared to classical optimal control methods for robot manipulation, RL requires less system knowledge for modeling and circumvents analytical intractability using approximations and a data-driven approach [2].
+However, research in multi-agent RL for multi-robot control is still in its infancy because of the following challenges:
+ 1. **Stability of the learning process.**
+ This issue arises because an agent's observed transition function is unstable when its counterparts' policies are not stationary.
+ Instead of parallelizing the single-agent RL model, we should consider how to coordinate the robot-robot interaction as it determines the success or failure of policy learning.
+ 2. **Alignment of individual objectives to the system-wide objective.**
+ Autonomy in multi-robot systems has to entail not just the agents' amenability to coordinate actions with one another, but also the capability to make decisions based on the agents' personal requirements.
```

Comment:
replace amenability: you are not writing a poem, you want to get your ideas through with the reader